### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jamescridland**
Additional platforms and shifting Truefans to be in alphabetical order

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,14 +89,13 @@ The distributed nature of podcasting makes it complicated to link to a show/epis
 | Podbean             | ❌            | `https://podbean.com/media/share/dir-${uniqueEpisodeID}`                                          |
 | Podcast Addict      | ✅            | `https://podcastaddict.com/episode/${encodeURIComponent(enclosureURL)}`                           |
 | Podcast Guru        | ❌            | `https://app.podcastguru.io/podcast/${appleID}/episode/${slug}-${uniqueEpisodeID}`                |
-| TrueFans            | ❌            | `https://truefans.fm/${slug}/${uniqueEpisodeID}`                                                  |
 | Podfriend           | ❌            | `https://web.podfriend.com/podcast/${appleID}/${uniqueEpisodeID}`                                 |
-| Podknife            | ❌            | `https://podknife.com/episodes/${uniqueEpisodeID}`                                                |
```
Comment (nathangathright) [inline README.md:95]: Why do you want this removed? Episode links on the platform [still work for me.](https://podknife.com/episodes/introducing-baby-this-is-keke-palmer-585ba842-fa34-49f5-852e-78cb5db65b5f)

File: README.md [hunk 2]
```diff
@@ -110,10 +108,12 @@ These resources also support podcast foreign keys, but their primary purpose may
 | Odesli        | `https://pods.link/i/${appleID}`                                                                    |
 | Plink         | `https://plinkhq.com/i/${appleID}?to=page`                                                          |
 | pod.link      | `https://pod.link/${appleID}` <br> `https://pod.link/${base64url(feedURL)}`                         |
+| podcastdetails | `https://podcastdetails.com/?url=${feedURL}`                                                       |
```
Comment (nathangathright) [inline README.md:111]: Even though the logo stylizes the name as `podcastdetails`, it’s referred to as "Podcast Details" by the title tag and social sharing pages.

File: README.md (hunk 1 above)
Comment (jamescridland) [reply README.md:95]: Ah, you're right. I got confused between Podknife and Podvine (the latter of which has closed). I guess I'll go and revert that bit

File: README.md (hunk 2 above)
Comment (jamescridland) [reply README.md:111]: Roger that. I'll fix.

**Pull request closed without merging**